tasks/task01/narcissistic_numbers.py

```diff
@@ -0,0 +1,19 @@
+def countDigits(n):
+    count = 0
+    while n != 0:
+        n //= 10
+        count += 1
+    return count
+
+
+narcissistic_numbers = []
+for i in range(1, 1001):
+    number_of_digits = countDigits(i)
```

Comment:
I think it needs to be moved in function and called function in the loop.

For example, we will need to use this code many times or we want to test this code.